### DISCUSSION TITLE
需要reality 自己偷自己的教程

### DISCUSSION THREAD

**Discussion by 404gods** [XTLS/Xray-core#2306] [Q&A]
最近刚上了VLESS+REALITY+ xtls-rprx-vision，目标网站选的微软，看到大家都认为偷自己比较好，可是没有找到系统完整的教程，请问哪里有？或者哪位大佬百忙之中出一个小白教程。
还有个问题，我的vps上本身就有个网站，如果想和reality共用443端口该怎么操作？同求小白教程

**Comment by PaperPlaneDeemo** [comment]
https://github.com/chika0801/Xray-examples/tree/main/VLESS-XTLS-uTLS-REALITY/steal_yourself
https://github.com/XTLS/Xray-core/discussions/2246

**Reply by 404gods** [reply]
谢谢！具体步骤有什么教程吗？

**Reply by PaperPlaneDeemo** [reply]
互联网上目前好像还没有系统的教程，我也是自己一步步摸索出来的

**Reply by 404gods** [reply]
我用X-UI搭建的，更不知道该怎么搞了

**Reply by chika0801** [reply]
建议你去ytb搜下有不有手把手教程。至于文章类我也没看到有人写很入门级的。楼上发的链接是配置示例，用于手动安装时，复制粘贴配置的，你要有一些手动安装经验，和nginx基础使用知识才能用。太入门的文章真没有。互联网都靠自己学自己悟和现在问AI。
